app/Http/Transformers/PostTransformer.php

```diff
@@ -30,12 +30,15 @@ public function transform(Post $post)
             'signup_id' => $post->signup_id,
             'northstar_id' => $post->northstar_id,
             'media' => [
-
                 'url' => $post->getMediaUrl(),
                 'original_image_url' => $post->url,
                 'caption' => $post->caption,
             ],
             'tags' => $post->tagSlugs(),
+            'reactions' => [
+                'reacted' => $post->reactions->isNotEmpty() ? true : false,
```

Comment:
I think this already returns a boolean? So you probably don't need to set it with the ternary.  https://laravel.com/docs/5.4/collections#method-isnotempty

Reply:
yup great call!!!